pkg/libvmi/architecture.go

```diff
@@ -25,6 +25,6 @@ func RegisterArchitecture(arch string) {
 	architecture = arch
 }
 
-func isARM64() bool {
+func IsARM64() bool {
```

Comment:
Non sono sicuro che l'architettura debba essere spostata con libvmi.
Forse dovrebbe rimanere con factory

Reply:
You are right.

Done